src/tests/functional/plugin/shared/include/behavior/infer_request/set_blob_by_type.hpp

```diff
@@ -42,8 +43,7 @@ class InferRequestSetBlobByType : public testing::WithParamInterface<InferReques
         // Skip test according to plugin specific disabledTestPatterns() (if any)
         SKIP_IF_CURRENT_TEST_IS_DISABLED()
         APIBaseTest::SetUp();
-        std::shared_ptr<ngraph::Function> function = ngraph::builder::subgraph::makeConvPoolRelu(
-                {4, 3, 6, 8}, ngraph::element::Type_t::u8);
+        std::shared_ptr<ngraph::Function> function = ov::test::utils::make_conv_pool_relu({4, 3, 6, 8}, ov::element::u8);
```

Comment:
`ngraph::Function` -> `ov::Model`

Reply:
In scope of this PR I change only subgraph_builder related code